src/utils/decorators/input-label/input-label.js

```diff
@@ -236,6 +236,8 @@ const InputLabel = (ComposedComponent) => {
       const inputProps = super.inputProps || {};
 
       // set id so label will focus on input when clicked
+      // this decorator is scheduled for removal that's why I'm ignoring next line
```

Comment:
I think that comment should be impersonal, e.g.:
`// test ignored, this decorator is scheduled for removal`

Reply:
It's about to removed at all in a week maybe, you want me to change it?